docs/workflow/pycbc_make_coinc_search_workflow.rst

```diff
@@ -685,23 +685,6 @@ Note that the ``LOGICAL_FILE_NAME`` for the veto files is just the name of the
 file, but for the two inspiral files it contains the subdirectory that the
 workflow uses to organize the files by GPS time. In the case of this file Pegasus will delete from the workflow the jobs that create the files ``H1-VETOTIME_CAT3-1169107218-1066800.xml``, ``L1-VETOTIME_CAT3-1169107218-1066800.xml``, ``116912/H1-INSPIRAL_FULL_DATA_JOB0-1169120586-1662.hdf``, and ``116912/H1-INSPIRAL_FULL_DATA_JOB1-1169120586-1662.hdf`` when it plans the workflow. Insted, the data will be re-used from the URLs specified in the cache. Since ``site="local"`` for these files, Pegasus expects that the files all exist on the host where the workflow is run from.
 
-To re-use data from a remote cluster, the URLs must contain a file transfer
-mechanism and the ``SITE`` should be set to ``remote``. For example, if the
-files listed in the example above are available on
-``sugwg-condor.phy.syr.edu`` and you want to re-use them in a workflow on the
-AEI Atlas cluster, then the cache file would contain::
-
-    H1-VETOTIME_CAT3-1169107218-1066800.xml gsiftp://sugwg-condor.phy.syr.edu/home/dbrown/projects/aligo/o2/analysis-4/o2-analysis-4/output/results/1._analysis_time/1.01_segment_data/H1-VETOTIME_CAT3-1169107218-1066800.xml pool="remote"
```

Comment:
Examples _may_ be good to retain?

Reply:
I don't know how we are supposed to do this anymore with scitokens. Perhaps we would just use `scp` or `ftp`?? I'll check the relevant syntax.